jenkins/bootstrap.py

```diff
@@ -832,13 +832,13 @@ def bootstrap(args):
     exc_type = None
 
     try:
+        setup_magic_environment(job)
```

Comment:
oooo yeah... hmm should I just make sure we run this before the "setup_root" then? https://github.com/jessfraz/test-infra/blob/47a474401d1fe0d18616b4550dc9ec2a0eccddf4/jenkins/bootstrap.py#L687